library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -53,7 +57,8 @@ impl Command {
 
         let pid = unsafe {
             match result {
-                0 => {
+                0 => (#[unwind(aborts)]
```

Comment:
This would be the first usage of `#[unwind(aborts)]` outside a test, and the first usage of `#[unwind]` on a closure. Not sure if this is stable enough.

(I believe this attribute was originally only meant for `extern` functions.)

@Mark-Simulacrum Do you know, or do you know who would know?

Reply:
Thanks for the review.

In defence of using this here: my MR adds a test case which exercises this panic -> unwind -> abort path, and of course there is no non-abort return path because the closure ends with `_exit` (and returns `!`)

If `#[unwind(aborts)]` is not stable enough, should I open-code something with `catch_unwind`?  It seemed to me that using a built-in library facility (even an unstable one) for this was better than an ad-hoc reimplementation of the same functionality.